index.html

```diff
@@ -1,8 +1,75 @@
 <!doctype html>
 <html>
-<head>
-</head>
-<body>
-<script type="text/javascript" src="AlbertMineSweeper.script.js"></script>
-</body>
-</html>
\ No newline at end of file
+	<head>
+		<title>AlbertMineSweeper Board</title>
+		<link rel="stylesheet" type="text/css" href="ams_board.css">
+		<script type="text/javascript" src="/bower_components/jquery/dist/jquery.js"></script>
+		<script type="text/javascript" src="ui.js" id="USER_INTERFACE"></script>
+	</head>
+
+	<body>
+		<h1>Albert Mine Sweeper</h1>
+		<h4>Project Status:</h4>
+		<ul>
+			<li>Current: End Conditions</li>
+			<li>Next: Adjustable Game</li>
+			<li>Next: User Interface for:
+				<ol>
+					<li>Resetting Game</li>
+					<li>Adjusting Game Settings</li>
+				</ol>
+			</li>
+			<li>Next: Clock</li>
+			<li>Next: Match Graphics to MS Minesweeper?</li>
+			<li>Complete:
+				<ul>
+					<li>Game Logic</li>
+					<li>GitHub/Sublime Setup</li>
+					<li>Code Refactoring</li>
+					<li>Reveal Squares when Clicked</li>
+					<li>Right-Click Flags</li>
+				</ul>
+			</li>
+		</ul>
+		<div class="base">
+			<div class="row"> 
+				<!--The "." was needed, or else the margins in some of the cells became 
+				unbalanced when some cells were visted and others not yet visited.-->
+				<div class="cell" data-row="0" data-col="0">.</div>
```

Comment:
Something I learned recently is `display: flex;`. If you use display flex you won't need these ".".

1. Remove the dots
2. Add the following css to your ".row" class
  ```css
        display: flex;
  ```
3. Add the following css to your ".cell" class
  ```css
	display: flex;
	align-items: center;
	justify-content: center;
  ```

This will center the content of each cell horizontally and vertically.